src/utils.ts

```diff
@@ -0,0 +1,25 @@
+// Copyright 2019 Diffblue Limited. All Rights Reserved.
+
+import { ILooseTestData, ITestData } from './test';
+
+/**
+ * Function whose signature claims to return a given type, but throws an error. Used to satisfy types during conversion from loose test to strict test
```

Comment:
Slightly rephrase and split over multiple lines:
```suggestion
 * Function whose signature claims to return a given type, but in practise always just throws an error.
 * Used to satisfy types during conversion from loose test to strict test
```